final_task/rss_reader/rss_item.py

```diff
@@ -0,0 +1,36 @@
+import html
+import json
+from dataclasses import dataclass
+
+
+@dataclass
+class RssItem:
+    '''
+    Represents a single news from RSS channel
+
+    source: stores link to rss channel of the news
+    date: stores date on YYYY%MM%DD format
+    '''
+    title: str
+    published: str
+    link: str
+    media: str
+    source: str
+    date: str
+
+    @classmethod
```

Comment:
Why did you need to make this a `classmethod`?

Reply:
This is a factory method which is called to create an instance of the class from dict

Reply:
Why can't you just use RssItem(**item)? You dont really need this method